examples/plot_simulate_alpha.py

```diff
@@ -46,7 +46,13 @@
     'sigma_t_evdist_1': 6.0,
     'input_dist_A_weight_L2Pyr_ampa': 5.4e-5,
     'input_dist_A_weight_L5Pyr_ampa': 5.4e-5,
-    'sync_evinput': 1
+    'sync_evinput': 1,
+    "prng_seedcore_evdist_1": 3,
+    "prng_seedcore_evprox_1": 3,
+    "prng_seedcore_evprox_2": 3,
+    "prng_seedcore_extgauss": 3,
+    "prng_seedcore_extpois": 3,
+    "prng_seedcore_input_dist": 3
```

Comment:
do you need to set all of them?

Reply:
Since the conductances for the evoked inputs are turned off, I guess not. It should be noted, however, that without setting these, a user who decides to restore the evoked input conductances (or e.g. turn on the poisson input) will notice that hnn-core behavior will diverge from that of HNN.

Reply:
but you updated the default.json? They have different seeds?

Reply:
Yes, default.json has been updated to reproduce the exact dynamics of the 'default' param set in HNN, however, recall that stabilizing the gid and seed assignment will cause hnn-core simulations to use different seeds than HNN whenever a rhythmic input is added or subtracted.

Reply:
I could modify the gamma_L5weak_L2weak.json param file used in plot_simulate_gamma.py as well, but this would require setting `{'prng_seedcore_extpois': -3}` within the json file, which seems questionable to me. Do we know why this parameter was set to a negative number to begin with?